sensor/kubernetes/sensor/deployment_identification.go

```diff
@@ -35,7 +36,7 @@ func populateFromServiceAccountTokenFile(out *storage.SensorDeploymentIdentifica
 		return errors.Wrapf(err, "reading token from file %s", tokenFile)
 	}
 
-	saToken, err := jwt.ParseSigned(string(bytes.TrimSpace(tokenBytes)))
+	saToken, err := jwt.ParseSigned(string(bytes.TrimSpace(tokenBytes)), []jose.SignatureAlgorithm{jose.RS256})
```

Comment:
Any reason to restrict the signature algorithm here compared to using any algorithm and the new `ParseSigned` function from the internal `jwt` package ?

Reply:
Good catch!